From 24dbf8eb0832876f4c30e71ec8a1ce95d2afc39d Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Thu, 22 Apr 2021 22:53:04 +0530
Subject: [PATCH] Fix vendor_ramdisk naming convention

vendor_boot ramdisk name has been changed to vendor_ramdisk
and also vendor-boot to venfor_boot.
Changed the same.

Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 groups/vendor-boot/true/AndroidBoard.mk | 4 ++--
 groups/vendor-boot/true/files.spec      | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/groups/vendor-boot/true/AndroidBoard.mk b/groups/vendor-boot/true/AndroidBoard.mk
index b3a9883..1d67c93 100644
--- a/groups/vendor-boot/true/AndroidBoard.mk
+++ b/groups/vendor-boot/true/AndroidBoard.mk
@@ -1,5 +1,5 @@
-BOARD_PREBUILT_VENDOR_BOOT_DIR := $(TARGET_DEVICE_DIR)/vendor-boot-ramdisk
-INTERNAL_VENDOR_RAMDISK_TARGET := $(call intermediates-dir-for,PACKAGING,vendor-boot)/vendor-ramdisk.cpio.gz
+BOARD_PREBUILT_VENDOR_BOOT_DIR := $(TARGET_DEVICE_DIR)/vendor_boot-ramdisk
+INTERNAL_VENDOR_RAMDISK_TARGET := $(call intermediates-dir-for,PACKAGING,vendor_boot)/vendor_ramdisk.cpio.gz
 
 .PHONY: vendor_boot_prebuilt
 vendor_boot_prebuilt:
diff --git a/groups/vendor-boot/true/files.spec b/groups/vendor-boot/true/files.spec
index 9afffaf..aa3ec29 100644
--- a/groups/vendor-boot/true/files.spec
+++ b/groups/vendor-boot/true/files.spec
@@ -1,5 +1,5 @@
 [devicefiles]
-{{target}}/vendor-boot-ramdisk: vendor-boot-ramdisk
+{{target}}/vendor_boot-ramdisk: vendor_boot-ramdisk
 
 [mapping]
-{{target}}/vendor-boot-ramdisk: vendor-boot-ramdisk
+{{target}}/vendor_boot-ramdisk: vendor_boot-ramdisk
-- 
2.17.1

